app/src/pages/access/AccessRequestPage.tsx

```diff
@@ -142,8 +143,7 @@ export const AccessRequestPage: React.FC = () => {
   let initialValues: any;
   let validationSchema: any;
   let requestForm: any;
-
-  if (keycloakWrapper?.getIdentitySource()?.toLowerCase() === 'bceid') {
+  if (keycloakWrapper?.getIdentitySource()?.toUpperCase() === SYSTEM_IDENTITY_SOURCE.BCEID) {
```

Comment:
Do we need the `toUpperCase()`, since that always returns either our enum or null?